movie-app-code-review-main/src/app/components/modal/modal-window.ts

```diff
@@ -0,0 +1,79 @@
+import { BaseComponent } from '@components/base-component';
+import { MyfavoriteComponent } from '@components/button/button';
```

Comment:
js uses CamelCase when creating names of functions, variables, etc.
It would be correct to write MyFavoriteComponent